### DISCUSSION TITLE
Design Notiz von einem Freund

### DISCUSSION THREAD

**Discussion by Maxman002** [AuDojo/AuDojo#84] [UI Design]
# Design Übersicht

## 4 Arten von Grafikdesign

- Gestaltung von Zeichen
- Mediendesign
- Corporate Design
- Werbegrafik

## Prinzipien guten Designs

- Innovation: Gutes Design ist: "Most advanced, yet acceptable."
- Benutzbarkeit: Gutes Design ist Intuitiv. "Don't make me think"
- Authentizität: Gutes Design hebt den Kern hervor, statt ihn zu täuschen.
- Stringenz: Gutes Design ist Konsequent und aufeinander bezogen.
- Reduzierung: Keep it simple and stupid, weniger hilft mehr.
- Fokussierung: Klare Hierarchien, welches Element hat die größte Bedeutung?
- Begründbarkeit: Jedes Element muss einen Sinn verfolgen.
- Regeln: Regeln helfen schneller Ästhetik zu finden.
- Kompromisse: Manchmal müssen Regeln gebrochen werden. 

## Schreib- und Leserichtung

Wer im Foto von Links nach Rechts ( -> ) geht, geht in die Welt hinaus (Gefühl von Abenteuer, Aufbruch, Abwechslung, Aufregung) und wer von Rechts nach Links ( <- ) geht, der kommt nach Hause (Gefühl von Geborgenheit, Vertrauen, Sicherheit, Heimkehr).

Links ist der Start, Rechts ist das Ende.

Wenn eine Person auf einem Foto nach Links guckt, dann blickt sie zurück (in die Vergangenheit), wenn sie nach Rechts guckt, dann blickt sie nach vorne (in die Zukunft).

## Gesetze

- Gesetz der Schreib- und Leserichtung
- Gesetz der durchgehenden Linie
- Gesetz der Kontinuität
- Gesetz der guten Richtung (Horizontal/Vertikal, nicht quer)
- Gesetz der Ähnlichkeit
- Gesetz der Nähe
- Gesetz der gemeinsamen Region
- Gesetz der guten Gestaltung
- Gesetz der Erfahrung
- Gesetz der Prägnanz
- Hicksches Gesetz (Einfachheit)
- Gesetz des Kontext
- Gesetz der Eindeutigkeit

## Notizen

- Horizontale Linien wirken beruhigend und schaffen Weite
- Vertikale Linien wirken begrenzend
- Diagonale Linien wirken dynamisch

- Linien leiten den Blick
- Punkte halten den Blick
- Flächen sind Grundlage und Umfeld

## Optische Bündigkeit

Rundungen (Kreise etc.) und Spitzen müssen die Grundlinie bzw. die X- oder Versalhöhe ein wenig durchbrechen, um optisch bündig zu wirken.

## Optische Mitte

Vertikal mittige Elemente werden oft als zu weit unten wahrgenommen. Deshalb werden vertikale Elemente (z.B. der Mittelstrich der Buchstaben "H" oder "E") oftmals ein weniger Höher platziert.

Hohe Elemente wirken leichter, tiefe Elemente wirken schwerer.
Horizontale Linien wirken stärker, senkrechte Linien wirken leichter. Deshalb werden bei Schriften horizontale Linien immer ein wenig schmaler abgedruckt (z.B. bei "H" oder "E")

Zentrierung bei asymetrischen Satzanfängen und -enden sollte per Hand und nach Augenmaß funktionieren. Zur Not rechts oder linksbündig.

Harmonie und Anmutung von Formen und Figuren

Geometrische Anmutung -> Strenge Anmutung -> Lebendige Anmutung

Goldener Schnitt
Fibonacci Folge

## Gewichtung und Platzierung

- Gewichtung entsteht durch ein Zusammenspiel aus Anzahl, Größe, Tönung und Platzierung.
- Dunkle Farben wirken schwerer, helle Farben wirken leichter.
- Große Formen wirken schwerer, kleine Formen wirken leichter.
- Statisches Gleichgewicht oder dynamisches Schwergewicht.
- Symmetrie oder Asymmetrie.
- Symmetrische Formen zentrieren, Asymmetrische Formen Offset.

## Aufteilung und Platzierung

- Mit Rastern arbeiten (z.B. dritteln oder bei komplexen Layouts sechsspaltig).
- Gerade Aufteilung fördert Symmetrie und Ausgewogenheit.
- Teilung durch Ungerade fördert dynamische Layouts.

## Bildkomposition

- Gerader, mittiger Blick und Ausschnitt: Zentrieren.
- Seitlicher Ausschnitt: Seitlicher Versatz, am Rücken orientieren.

## Schriftarten:

- Antiqua: weiche, feine Serifen
- Egyptienne: fettere, härtere und kantigere Serifen
- Grotesk: Sans Serif Schriften (ohne Serifen)
- Schreibschriften: Zusammenhängende, kunstvolle Schriften
- Handschriften: Handschriftlich anmutende Schriften
- Display: Experimentelle, kunstvolle (Über-) Schriften

### Schriften kombinieren

Wichtiges Kriterium für Harmonie zwischen verschiedenen Schriften sind Strichstärkenunterschiede (sollten gleich sein) sowie X-Höhe.

Ligaturen

Ligaturen sind miteinander "verwachsene" Buchstaben bzw. deren Verbindungen (ff, fi, ch, ck etc.)

Schriftgrößen

Wichtig ist, dass die Schriftgrößen zueinander in Bezug stehen.

- Satzbreite: 70-90 Zeichen (oder 130mm bei Büchern, hängt mit dem Abstand zwischen Text und Auge zusammen)
- Zeilenabstand: Mindestens soviel wie Wortabstand (bisschen mehr)
- Wortabstand: Der Kegel eines Buchstaben
- Schriftgröße Web: min. 16px (bei einem Bildschirmabstand von 60cm)
- Schriftgröße Print: min. 9px (9-11px; bei einem Abstand von etwa 40cm)
- Blocksatz für sehr dichte Layouts (Zeitungen)
- Linksbündiger "Flattersatz" ist zu unruhig

## Farben

- Komplementärfarben: Auf dem Farbkreis gegenüberliegende Farbenpaare
- Dreiklang (Triade): Drei Farben gleichen Abstands auf dem Farbenkreis (120°)
- Vierklang: Vier Farben gleichen Abstands auf dem Farbenkreis (90°)
- Analog: Nebeneinander liegende Farben
- Monochrom: Ein Farbton mit verschiedenen Abstufungen

**Comment by anomatomato** [comment]
Wow danke sehr :open_mouth:
